crypto-ffi/bindings/swift/WireCoreCryptoUniffi/BuildSettings.xcconfig

```diff
@@ -0,0 +1,23 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+// Configuration settings file format documentation can be found at:
+// https://help.apple.com/xcode/#/dev745c5c974
+
+MARKETING_VERSION = 4.2.0
+CURRENT_PROJECT_VERSION = 1.0
```

Comment:
We have duplicate definitions of versions. Since we're always going to release the two packages together, it makes sense to have single definition. To that end, we could have a `BuildSettings.xcconfig` file in the parent dir, with these definitions, and then just `#include` them here.

Ideally, we would be able to tell (perhaps via `.pbxproj`?) to use the parent dir's config settings directly. In that case, we would not even need this file, nor the other neighboring file.

Reply:
Yeah I'll move the BuildSettings.xcconfig to root swift bindings folder and references it from both projects